router/router_test.go

```diff
@@ -153,6 +154,7 @@ func initRouter() {
 	config.Reset()
 	admin.Init()
 	logger.Reset()
+	misc.Init()
```

Comment:
is this really needed?

Reply:
There is a global variable in misc package which get's triggered to nil pointer exception whenever a tmpdir is created is what I saw in few tests